src/main/java/io/vertx/redis/client/impl/CommandReporter.java

```diff
@@ -23,7 +23,11 @@ enum Tags {
     DB_USER("db.user", reporter -> reporter.user),
     DB_INSTANCE("db.instance", reporter -> reporter.database),
     DB_STATEMENT("db.statement", reporter -> reporter.command),
-    DB_TYPE("db.type", reporter -> "redis");
+    DB_TYPE("db.type", reporter -> "redis"),
+    NETWORK_PEER_ADDRESS("network.peer.address", reporter-> reporter.networkPeerAddress),
+    NETWORK_PEER_PORT("network.peer.port", reporter -> reporter.networkPeerPort),
+    SERVER_ADDRESS("server.address", reporter -> reporter.serverAddress),
+    SERVER_PORT("server.port", reporter -> reporter.serverPort);
```

Comment:
Maybe move these above, to the _Generic_ section?

Reply:
Yup